config/config.go

```diff
@@ -291,6 +292,12 @@ func New(logger *zap.SugaredLogger, osArgs []string) (config, error) {
 		return cfg, err
 	}
 
+	mainFS.StringVar(&cfg.Injector.LogLevel, "injector-log-level", "", "The LOG_LEVEL used for the injector pods")
```

Comment:
```suggestion
	mainFS.StringVar(&cfg.Injector.LogLevel, "injector-log-level", "DEBUG", "The LOG_LEVEL used for the injector pods")
```

Could we have a default when unspecified?

Reply:
Sure, I've made the change.